docs/source/overview/traffic_monitor.rst

```diff
@@ -56,3 +56,6 @@ Protocol Engagement
 -------------------
 Short polling intervals of both the :term:`cache servers` and Traffic Monitor peers help to reduce customer impact of outages. It is not uncommon for a :term:`cache server` to be marked unavailable by Traffic Monitor - in fact, it is business as usual for many CDNs. Should a widely requested video asset cause a single :term:`cache server` to get close to its interface capacity, the Health Protocol will "kick in," and Traffic Monitor marks the :term:`cache server` as unavailable. New clients want to see the same asset, and now :ref:`tr-overview` will send these customers to another :term:`cache server` in the same :term:`Cache Group`. The load is now shared between the two :term:`cache servers`. As clients finish watching the asset on the overloaded :term:`cache server`, it will drop below the threshold and gets marked available again, and new clients will begin to be directed to it once more. It is less common for a :term:`Delivery Service` to be marked unavailable by Traffic Monitor. The :term:`Delivery Service` thresholds are usually used for overflow situations at extreme peaks to protect other :term:`Delivery Services` in the CDN from being impacted.
 
+Handling Caches with same service ip
+-------------------
+Traffic monitor is able to direct traffic router to route traffic to groups of :term:`cache servers` with the same service ip address that is routed with qual cost multi path(ECMP) on the last hop, but not to individual :term:`cache server` within that group. To monitor :term:`cache servers` that within that group, http keep-alive header is omitted with the ``health.polling.keepalive`` and ``stat.polling.keepalive`` traffic monitor profile parameter so that the TM source port changes for those connections and the TM can get to all the servers within that group via ECMP. That it updates the responses using the via header. In the event of :dfn:`Health Protocol` determining the overall health of one of the :term:`cache servers` in a anycast group by evaluating network throughput and load against values configured in :ref:`to-overview` to exceed their limit, it declares the whole groups as unhealthy.
```

Comment:
Nits: `IP` and `HTTP` should be capitalized.

Reply:
Agreed on all counts, how should I expand on ECMP? It's a standard and TM is not really enforcing it in any way, not sure it needs further explanation.

Reply:
I guess I would just make it a link to [the ECMP wikipedia page](https://en.wikipedia.org/wiki/Equal-cost_multi-path_routing). That way if people wanna read about it that's up to them.

Reply:
I'm just going to delete the reference to ECMP, its not relevant.

Reply:
Oh, gotcha.